SA/design_concept/content/poc/options_ui_haskellgi.typ

```diff
@@ -2,17 +2,17 @@
 
 = Haskell-gi
 
-#ac("GTK") is a widget toolkit that allows the creation of #ac("UI")s that work on
+#ac("GTK") is a widget toolkit that allows the creation of #ac("UI")s that works on
```

Comment:
I think `work` is better in this case. `work` refers to `UIs`, indicating that the created UIs work on multiple OS. `works` refers to the `GTK toolkit` indicating that the toolkit works on multiple OS.

Reply:
agree 👍